lang/go/eval/symbol/map.go

```diff
@@ -58,8 +59,31 @@ type MapSymbol struct {
 	Map   map[string]Symbol `ko:"name=map"`
 }
 
-var _ Symbol = &MapSymbol{}
+var (
+	_         Symbol = &MapSymbol{}
+	mapGoType        = reflect.TypeOf(map[string]interface{}{})
+)
+
+// DisassembleToGo converts a Ko value into a Go value
+func (ms *MapSymbol) DisassembleToGo(span *model.Span) (reflect.Value, error) {
```

Comment:
See comment for Series. Same applies here as the map value type can be created accurately. Just a todo comment here suffices.